### DISCUSSION TITLE
Mute tweets by source

### DISCUSSION THREAD

**Discussion by robsonsobral** [robertodoering/harpy#724] [Ideas]
I would love to be able to mute tweets by source, so I could get my timeline free of:

- "I've sold a new video";
- "I've made X points on Y game";
- "See my Onlyfans page";
- and others apps which tweets on user behalf.

Thank you.

**Comment by robertodoering** [comment]
Tweets that are created automatically (e.g. when you share a youtube video or an in-game achievement) are indistinguishable from normal Tweets. So this won't be possible unfortunately.

**Reply by robsonsobral** [reply]
But the source is reported! I used this feature on... I don't remember the name of the app.

**Reply by robertodoering** [reply]
The "Source" of a Tweet refers to the platform that was used to Tweet (e.g. Twitter for Android, Twitter for iOS, harpy for Twitter, Tweetbot, etc.). You can see it in harpy by clicking on the Tweet.
When you share a tweet, you share it through a platform (e.g. through the official Twitter Android app), which will then be the source for the Tweet.
